python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -73,8 +98,8 @@ def optimize(self, model,
                  training_data,
                  validation_data=None,
                  metric=None,
-                 direction=None,
-                 cpu_num=None):
+                 cpu_num: int = None,
+                 trials: int = 100):
```

Comment:
`trails` may confuse our users with the "trial" in automl searching. Since this param is not used every time, we may make it longer and easier to understand such as `latency_sample_num`

Reply:
`direction` can be moved here and only for record, since this is a "feature" according to the metric, and no users would like to filter a model that is "worse" than xxx metric value.

Reply:
done